README.md

```diff
@@ -74,7 +74,11 @@ Now it's your turn to write SQL querys to achieve the following results:
 1. Count the total number of states in each country.
 
 ```
-Your query here
+SELECT c.name, COUNT(s.*)
+FROM countries c
+INNER JOIN states s
+	ON c.id = s.country_id
+GROUP BY c.name;
```

Comment:
You should use `LEFT JOIN` in here, by using `INNER JOIN` you are leaving out any countries that do not have any states and they won't show up in the list